src/Entity/Announcement.php

```diff
@@ -42,6 +42,11 @@ class Announcement
    */
   private $text;
 
+  /**
+   * @ORM\Column(type="string", length=255)
```

Comment:
The column type here *shouldn't* be string.  This should be a reference to a Category entity.  See line 31 as an example for creating relationships between entities.  You'll also need to edit the Category entity to reflect this change.